native_client/ctcdecode/Makefile

```diff
@@ -4,6 +4,11 @@ include ../definitions.mk
 
 NUM_PROCESSES ?= 1
 
+# ARM64 can't find the proper libm.so without this
+ifeq ($(TARGET),rpi3-armv8)
+LDFLAGS_NEEDED += $(RASPBIAN)/lib/aarch64-linux-gnu/libm-2.24.so
```

Comment:
Even if it's not very nice, we should avoid adding more unniceness with hard-coding the version. `libm.so.6` should be a good stable symlink